File: sdi3-180.Cli-SOAP-CSharp/sdi3-180.Cli-SOAP-CSharp/ListarUsuarios.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace sdi3_180.Cli_SOAP_CSharp
{
    class ListarUsuarios
    {
        private EjbAdminServiceService adminService = new EjbAdminServiceService();

        public void execute(){
	        user[] users = adminService.getUsuarios();
	        Console.WriteLine("ID - Nombre - Apellidos - Login - Password - Email - Estado - Viaje promovidos - Viajes participado");
	        int viajePromovido = 0;
	        int viajeParticipado = 0;
            string format = "{0,-15} {1, -15} {2, -15} {3, -15} {4, -15} {5, -15} {6, -15} {7, -15} {8, -15}";
	        foreach (user u in users) {
		        viajePromovido = adminService.getViajesPromovido(u);
		        viajeParticipado = adminService.getViajesParticipado(u);
                string line = String.Format(format, 
				        u.id, u.name, u.surname, 
				        u.login, u.password, 
				        u.email, u.status.ToString(),
				        viajePromovido, viajeParticipado);
                Console.WriteLine(line);
	        }	
        }

    }
}

File: sdi3-180.Cli-EJB/src/com/sdi/client/accion/ListarUsuarios.java
package com.sdi.client.accion;

import java.util.List;

import com.sdi.business.AdminService;
import com.sdi.business.impl.RemoteEjbServicesLocator;
import com.sdi.model.User;

public class ListarUsuarios{

	private AdminService adminService = new RemoteEjbServicesLocator().getAdminService();

	public void execute(){
		List<User> users = adminService.getUsuarios();
		System.out.println("ID - Nombre - Apellidos - Login - Password - Email - Estado - Viaje promovidos - Viajes participado");
		int viajePromovido = 0;
		int viajeParticipado = 0;
		for (User u: users) {
			viajePromovido = adminService.getViajesPromovido(u);
			viajeParticipado = adminService.getViajesParticipado(u);
			System.out.printf("%d - %s - %s - %s - %s - %s - %s - %d - %d\n", 
					u.getId(), u.getName(), u.getSurname(), 
					u.getLogin(), u.getPassword(), 
					u.getEmail(), u.getStatus().toString(),
					viajePromovido, viajeParticipado);			
		}	
		
	}

}
